buddyboss/buddyboss-platform
# PROD-784 Updating learndash group changes privacy settings on social group

### All Submissions:
https://buddyboss.atlassian.net/browse/PROD-4905






### Changes proposed in this Pull Request:



Fixes #2618  .

### How to test the changes in this Pull Request:

1. Make sure that Social Group Sync and LearnDash Group Sync are enabled. Please see https://i.ibb.co/4VcQzhs/p1.png.
2. Create a Hidden Social Group
3. Create a LearnDash Group and associate it to the Social group you just created. https://i.ibb.co/HKsY8cD/p2.png
4. Try to add some users to the LearnDash group. https://i.ibb.co/7KRKhvG/p3.png
5. Now go to the Social Group and notice that it is now Hidden as it was.

### Proof Screenshots or Video

https://www.loom.com/share/a8a3b1dc05d549b3bc3532280d670f97

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
